.github/workflows/release.yml

```diff
@@ -83,18 +93,19 @@ jobs:
         uses: juliangruber/read-file-action@v1
         with:
           path: ./version.txt
-          
+      
       - name: Create Release
         uses: actions/create-release@v1
+        id: create_release
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          tag_name: ${{ github.ref }}
+          tag_name:  ${{ github.ref }}
           release_name: Release ${{ steps.version.outputs.content }}
           body: |
-            ## Compiled Binaries
-            - Windows: https://storage.googleapis.com/${{ secrets.GCP_BUCKET }}/${{ steps.version.outputs.content }}/windows-latest/go-spacemesh.exe
-            - Linux: https://storage.googleapis.com/${{ secrets.GCP_BUCKET }}/${{ steps.version.outputs.content }}/ubuntu-latest/go-spacemesh
-            - Mac: https://storage.googleapis.com/${{ secrets.GCP_BUCKET }}/${{ steps.version.outputs.content }}/macos-latest/go-spacemesh
```

Comment:
It will be nice to change this to `macOS` from `Mac` and order releases by platform popularity: Windows, macOS, Linux .

Reply:
Done